twemproxy/tests/compose/docker-compose.yaml

```diff
@@ -0,0 +1,48 @@
+version: '2'
+services:
+
+  redis1:
+    image: redis
+    ports:
+      - "6101:6379"
+
+  redis2:
+    image: redis
+    ports:
+      - "6102:6379"
+
+
+  etcd0:
+    image: quay.io/coreos/etcd:v2.2.0
+    volumes:
+      - /usr/share/ca-certificates/:/etc/ssl/certs
+    ports:
+       - 4001:4001
+       - 2380:2380
+       - 2379:2379
+    command: |
+            -name etcd0
+            -advertise-client-urls http://0.0.0.0:2379,http://0.0.0.0:4001,http://etcd0:2379,http://etcd0:4001
+            -listen-client-urls http://0.0.0.0:2379,http://0.0.0.0:4001
+            -initial-advertise-peer-urls http://0.0.0.0:2380
+            -listen-peer-urls http://0.0.0.0:2380
+            -initial-cluster-token etcd-cluster-1
+            -initial-cluster etcd0=http://0.0.0.0:2380
+            -initial-cluster-state new
+
+  twemproxy:
+    image: jgoodall/twemproxy
+    ports:
+      - 6100:6100
+      - 6222:6222
+    volumes:
+      - ${WAIT_FOR_IT_SCRIPT_PATH}:/wait-for-it.sh
+      - ${SETUP_SCRIPT_PATH}:/setup.sh
```

Comment:
Can we do these in Python?

Reply:
Any particular reason? The reason the wait-for-it is a shell script is because not all containers have python in them. It's a complex utility that I don't want to rewrite in python.

The second one I think is honestly simpler as a shell script than as a python script.

Reply:
These are run in the container, we can't run them outside the container

Reply:
I rewrote one in python but I think the other one is pretty good in shell. while for this one it has python in it, it's not always the case, and that script is used in a number of checks.